docs/GettingStartedDocs/OP-TEE/Hardware/ScalysTrustBox.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
need to fill in this TBD when you update this PR again

Reply:
Filled.